Cargo.lock

```diff
@@ -7056,21 +7056,22 @@ dependencies = [
 
 [[package]]
 name = "risc0-build-ethereum"
-version = "1.4.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "c39b6a66fa37ac33a97ee48dff3cbcaf7c842d4b859bf383d0c5b2c4f8bb5b48"
+version = "1.4.2"
```

Comment:
Assuming there won't be conflicts with the aggregation git dep using old version